lib/ship.rb

```diff
@@ -0,0 +1,23 @@
+class Ship
+  attr_reader :name,
+              :length,
+              :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = length
+  end
+
+  def hit
+    @health = (health - 1)
+  end
+
+  def sunk?
+    if @health >= 1
+      false
+    elsif @health <= 1
+      true
```

Comment:
This can be simplified to
```
def sunk?
  @health <= 0
end
```
Since the comparison will return true or false without using the if/elsif conditionals